### DISCUSSION TITLE
Control of program after call to sub OnFrameLoadEnd in Visual Basic

### DISCUSSION THREAD

**Discussion by Tetewind** [cefsharp/CefSharp#4637] [General]
Hello with Visual Basic , I load a webpage with CefSharp (_browser = New ChromiumWebBrowser("url") , after I read data in Sub  OnFrameLoadEnd , how do i regain control after calling to call Sub OnFrameLoadEnd?

**Comment by amaitland** [comment]
> how do i regain control after calling to call Sub OnFrameLoadEnd?

I'm not sure what you mean by this.

Would the answer to your question be the same if you were using a `non-CefSharp` specific event? If yes then https://stackoverflow.com/ would be a better place to ask generic programming questions.
